courses/models.py

```diff
@@ -61,3 +70,14 @@ class Module(models.Model):
 
     def __str__(self):
         return self.title
+
+    def to_webhook(self):
+        """
+        Webhook serialization
+        """
+        return {
+            'title': self.title,
+            'uuid': str(self.uuid),
+            'subchapters': self.subchapters,
+            'course_pk': self.course_id,
```

Comment:
Can this be `course_uuid` instead? The course id won't be useful outside of this database